### PR TITLE
Update Empower Her X Non Tech in Tech.md

### PR DESCRIPTION
**Pull request by Amytex**
My first pull request

### CODE REVIEW

File: Empower Her X Non Tech in Tech.md [hunk 1]
```diff
@@ -46,4 +46,5 @@ Learn more [here](https://twitter.com/empowerhercom/status/1554066239047274498?s
 
 - [Teniola Olowookere](https://github.com/Teniola-theDev) [My Experience at the EHC X NTIT Technical Writing Bootcamp](https://teniolaxyz.hashnode.dev/my-experience-at-the-empower-her-community-x-non-tech-in-tech-bootcamp)
 
-- [Halimat Chisom](https://github.com/Chisomgold): [My Experience at the EHC X NTIT Technical Writing Bootcamp](https://medium.com/@gearthdexter/bootcamp-experience-with-empower-her-x-non-tech-in-tech-technical-writing-de590524c2ce)
\ No newline at end of file
+- [Halimat Chisom](https://github.com/Chisomgold): [My Experience at the EHC X NTIT Technical Writing Bootcamp](https://medium.com/@gearthdexter/bootcamp-experience-with-empower-her-x-non-tech-in-tech-technical-writing-de590524c2ce)
+- [Mbah Uzoma What I learned at The Empower her Community and Non-Tech in Tech Bootcamp] (https://amytex5.hashnode.dev/what-i-learned-during-the-empower-her-community-and-non-tech-in-tech-bootcamp)
```
Comment (Bennykillua) [inline Empower Her X Non Tech in Tech.md:50]: Hi @Amytex  there is a mistake in your markdown syntax

Comment (Bennykillua) [reply Empower Her X Non Tech in Tech.md]: I would fix it from my end.

**Comment by Bennykillua** [comment]
Thanks for your contribution @Amytex

**Comment by Amytex** [comment]
Good morning Mr. Benny,

Thank you for your correcting my mistake, I do appreciate it.

I would like to know what I missed because I noticed there was a mistake
earlier but don't know how to fix it.

Thanks,
From Mbah Uzoma.

**Comment by Bennykillua** [comment]
Hi Mbah, 

It is actually nothing.  

### To embedd a link in markdown. 

Here was the mistake

```
[Google] (https://www.google.com/)

```

but is the correct syntx was 

```
[Google](https://www.google.com/)

```

There shouldnt be a space between the "]" and the next "("

Looking forward to more contribution from you @amytex

**Comment by Amytex** [comment]
Ooh!!
Thank you for the clarification.

**Pull request merged**